ansible/hetzner-single-deploy.yml

```diff
@@ -134,6 +134,15 @@
     virt_net:
       name: wirebox
       autostart: yes
+  - name: disable dnsmasq for public interface
+    lineinfile:
+      path: /etc/dnsmasq.d/disable_public_resolv
+      line: "except-interface={{ ansible_default_ipv4.interface }}"
```

Comment:
Hm, I'm confused: Wouldn't it be better to explicitly name the interface to listen on `interface=...`? :thinking: 

But, I have to admit that I'm absolutely no expert in this area.

Reply:
It turns out we don't need dnsmasq at all, libvirt even sets a config in "/etc/dnsmasq.d-available/libvirt-daemon" to disable listening on virbr0 interface.  I remember dnsmasq used to be a dependency for libvirt, but that was 7-8 years ago, and I only touched pure libvirt / KVM hosts again here at Wire. My bad!

This PR now removes dnsmasq altogether.